modular-particle-system/src/generators/pathGenerator.ts

```diff
@@ -0,0 +1,208 @@
+import { Position } from "../types";
+import { Particle } from "../particle";
+import { ParticleGenerator } from "./generator";
+import { lerp } from "../utilities";
+import { ModuleObject } from "../module";
+import { ParticleEffect } from "../particleEffect";
+import { loadSerializedProperty, deserializePrimitiveDataType } from "../serialization/moduleSerialization";
+
+/**
+ * Generator module that creates in a path.
+ *
+ * @module
+ * interval {
+ *      @tooltip        TODO
+ *      @type           Number
+ *      @min            0
+ *      @step           0.01
+ *      @defaultValue   0.1
+ * }
+ * p1 {
+ *      @tooltip        TODO
+ *      @type           Position
+ * }
+ * p2 {
+ *      @tooltip        TODO
+ *      @type           Position
+ * }
+ * p3 {
+ *      @tooltip        TODO
+ *      @type           Position
+ * }
+ * p4 {
+ *      @tooltip        TODO
+ *      @type           Position
+ * }
+ * p5 {
+ *      @tooltip        TODO
+ *      @type           Position
+ * }
+ * p6 {
+ *      @tooltip        TODO
+ *      @type           Position
+ * }
+ * padding {
+ *      @tooltip        TODO
+ *      @type           Number
+ * }
+ * between {
+ *      @tooltip        TODO
+ *      @type           Boolean
+ * }
+ */
+
+export class PathGenerator extends ParticleGenerator {
+    p1: Position = { x: 0, y: 0 };
+    p2: Position = { x: 0, y: 0 };
+    p3?: Position;
+    p4?: Position;
+    p5?: Position;
+    p6?: Position;
+    padding?: number;
+    between?: boolean;
```

Comment:
This is essentially same property as what Artem is currently developing at

https://github.com/Risto-Paasivirta/ParticleSystem/pull/91/files   ShapeGenerator.edgesOnly

I would like if they would have the same name and behaviour, which would mean changing `between` to `edgesOnly`, setting the default value to `false` and inverting its usage in the code (definition would become opposite).

Reply:
I'm pushing these changes shortly.